apis/Google.Cloud.Diagnostics.AspNetCore/Google.Cloud.Diagnostics.AspNetCore.IntegrationTests/Trace/TraceTest.cs

```diff
@@ -383,7 +383,7 @@ public void ConfigureServices(IServiceCollection services)
             {
                 c.BaseAddress = new Uri("https://google.com/");
             })
-            .AddHttpMessageHandler<UnchainedTraceHeaderPropagatingHandler>();
+            .AddGoogleOutgoingTraceHandler();
```

Comment:
Nit: let's indent this line and the three above.

Reply:
Done

Reply:
Done